docs/content/users/extend/additional-services.md

```diff
@@ -105,18 +105,31 @@ If you have add-ons that were installed before v1.22, update them with `ddev add
 
 ## Adding Custom Configuration to an Add-on
 
-Sometimes it's necessary to add custom configuration to an add-on. For example, in [`ddev-redis-7`](https://github.com/ddev/ddev-redis-7) the `image` is set to `image: redis:7.2-alpine`. If you wanted to change this to use `7.0-alpine` instead, you would have two choices:
+Sometimes it's necessary to add custom configuration to an add-on. For example, in [`ddev-redis`](https://github.com/ddev/ddev-redis) the `image` is set to `image: redis:${DDEV_REDIS_VERSION:-6-bullseye}`. If you wanted to change this to use `7.0-bookworm` instead, you would have two choices:
 
-1. Remove the `#ddev-generated` line in `docker-compose.redis-7.yaml` and edit the relevant line.
-2. Add a `.ddev/docker-compose.redis-7_extra.yaml` with the contents:
+1. With DDEV v1.23.5+, the override can be done using a special `.ddev/.env.*` file, where `*` is the name of the add-on (and its service), in this case `.ddev/.env.redis`:
```

Comment:
`something` in `.env.something` is only relevant if env variables are to be passed inside a service.

Otherwise the filename is irrelevant, it will be expanded by `docker-compose config` anyway.
I use the add-on name to name this `.env.*` file somehow.